deploy/lib/xquery/setup.xqy

```diff
@@ -4282,18 +4282,33 @@ declare function setup:get-configuration(
   $databases as xs:string*,
   $forests as xs:string*,
   $app-servers as xs:string*,
-  $user-ids as xs:unsignedLong*,
-  $role-ids as xs:unsignedLong*,
+  $users as xs:anySimpleType*,
+  $roles as xs:anySimpleType*,
   $mimetypes as xs:string*) as element()
 {
-  <configuration>
-    {setup:get-app-servers($app-servers)}
-    {setup:get-forests($forests)}
-    {setup:get-databases($databases)}
-    {setup:get-users($user-ids)}
-    {setup:get-roles($role-ids)}
-    {setup:get-mimetypes($mimetypes)}
-  </configuration>
+  let $user-configuration :=
+      typeswitch($users[1])
```

Comment:
Yup, I see that now, I had tested by injecting strings vs. integers from the Ruby capture_environment_config function directly which worked ok, however it does look like if the values are passed in as arguments they are quoted to strings. I will fix this shortly. Thanks!